services/README.adoc

```diff
@@ -6,4 +6,29 @@ The OpenShift Ansible Broker is used to provision Mobile Services to an OpenShif
 
 == Metrics Service
 
-The Metrics Service consists of Prometheus, with an OAuth Proxy for authentication and authorization. Mobile Services in your Project will automatically have metrics collected and available in Prometheus. You can configure your own custom services to also have metrics be collected by Prometheus. See the guide for link:making-mobile-services-discoverable-by-metrics.adoc[Making Services auto-discoverable by the Metrics Service] to do that.
\ No newline at end of file
+The Metrics Service consists of link:https://prometheus.io/[Prometheus] and link:https://grafana.com/[Grafana] with an OAuth Proxy
+tied to link:https://github.com/aerogear/mobile-docs/blob/master/services/giving-an-openshift-user-access-to-metrics.adoc[openshift permissions], 
+for authentication and authorization. It automates the gathering and display of service metrics by using features in Openshift, 
+link:https://github.com/aerogear/mobile-docs/blob/master/services/making-mobile-services-discoverable-by-metrics.adoc[annotations]
+and link:https://github.com/aerogear/mobile-docs/blob/master/services/working-with-grafana-dashboards.adoc[configMaps], 
+to allow the discovery of service metrics and visualisation of data. The metrics service makes it easy to monitor provisioned mobile services and 
```

Comment:
should we also mention device (SDK) metrics?

Reply:
I don't know enough about the device metrics work to speak about it. If there's something that you think would make sense to add I'll add it in?